portfolio/src/main/webapp/index.html

```diff
@@ -450,25 +478,44 @@ <h2 class="text-uppercase">Ahmaud Arbery</h2>
                   <h2 class="text-uppercase">Breonna Taylor</h2>
-                  <p class="item-intro text-muted">Lorem ipsum dolor sit amet consectetur.</p>
+                  <p class="item-intro text-muted">Her Murderers Still Run Free</p>
                   <img
                     class="img-fluid d-block mx-auto"
                     src="assets/img/martyrs/BreonnaTaylor.jpg"
                     alt="Breonna Taylor"
                   />
-                  <p>
-                    Use this area to describe your project. Lorem ipsum dolor sit amet, consectetur adipisicing elit.
-                    Est blanditiis dolorem culpa incidunt minus dignissimos deserunt repellat aperiam quasi sunt officia
-                    expedita beatae cupiditate, maiores repudiandae, nostrum, reiciendis facere nemo!
-                  </p>
+                  <h4>Justice for Breonna Taylor</h4>
                   <ul class="list-inline">
-                    <li>Date: January 2020</li>
-                    <li>Client: Finish</li>
-                    <li>Category: Identity</li>
+                    <li><b>Age:</b> 26</li>
+                    <li><b>Location of Death:</b> Louisville, Kentucky</li>
                   </ul>
+                  <p>
+                    A short time after midnight n March 13, 2020, three Louisville police officers Brett Hankison, Jon
+                    Mattingly, and Myles Cosgrove were sent to Breonna Taylor's home to execute a "no-knock" warrant.
+                    Upon arrival, the officers began to use a battering ram to enter Ms. Taylor's apartment, while she
+                    and her boyfriend were in bed. The loud banging led to Kenneth Walker, Ms. Taylor's boyfriend,
+                    grabbed his own weapon and fired the gun at the officers. The officers retaliated and shot Ms.
+                    Taylor
+                    <b>at least 8 times</b>. The officers were believed to have been investigating 2 men that were
+                    selling drugs far from Ms. Taylor's home and no drugs were found in her apartment. Hankison was
+                    fired from the Louisville Metro Police Department while Mattingly and Cosgrove have been placed on
+                    administrative reassignment. Currently, <b>none of the officers face any criminal charges</b>.
+                  </p>
+                  <hr />
+                  <h4>How to Help</h4>
+                  <p>
+                    <b>Donate Here: </b
+                    ><a class="interactive-pics" href="https://www.gofundme.com/f/9v4q2-justice-for-breonna-taylor"
+                      >Justice for Breonna Taylor (official)(#BREEWAYY)</a
+                    ><br />
+                    <b>Sign a Petition Here: </b
+                    ><a class="interactive-pics" href="https://www.change.org/p/andy-beshear-justice-for-breonna-taylor"
+                      >Justice for Breonna Taylor</a
+                    >
```

Comment:
Please fix formatting.

Reply:
These changes were merged from master so I didn't review them within this pull request. In addition, when I run make pretty it reformats the tags to be on the next line. Should we side with make pretty or should I manually fix them?

Reply:
Unless style guide permits (I can't find this style in it), we have to manually fix this. We can't rely on the formatting tool without manual review.

Reply:
When I manually fix the formatting, using the make pretty command changes it back to the odd formatting. Should we only use make validate in the future? Any use of make pretty will result in the odd formatting.